crowdfunding/users/views.py

```diff
@@ -0,0 +1,84 @@
+from django.http import Http404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status, generics, permissions
+
+from .models import CustomUser
+from .serializers import CustomUserSerializer, ChangePasswordSerializer
+from projects.permissions import IsOwnerOrReadOnly
+class CustomUserList(APIView):
+
+	permission_classes = [permissions.IsAuthenticatedOrReadOnly,
+	IsOwnerOrReadOnly]
+
+	# if request.user.is_authenticated:
+
+	def get(self, request):
+		users = CustomUser.objects.all()
+		serializer = CustomUserSerializer(users, many=True)
+		return Response(serializer.data)
+
+	def post(self, request):
+		serializer = CustomUserSerializer(data=request.data)
+		if serializer.is_valid():
+			serializer.save()
+			return Response(serializer.data)
+		return Response(serializer.errors)
+
+class CustomUserDetail(APIView):
+
```

Comment:
there's a lack of permissions here.
even not-logged-in users could delete other users?